torch/csrc/autograd/python_variable.h

```diff
@@ -7,7 +7,7 @@
 #include "torch/csrc/autograd/variable.h"
 
 // Python object that backs torch.autograd.Variable
-struct THPVariable {
+struct THP_CLASS THPVariable {
```

Comment:
this seems to have caused a problem -- do you need to include THP_export?

Reply:
I'm trying to fix the Windows build and failing. THNN.so is a separate library so symbols from _C.so have to be exported or something...